understanding/focus-appearance.html

```diff
@@ -53,389 +53,316 @@
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Make it easier to spot the keyboard focus.</dd>
+                  <dt>目標</dt>
+                  <dd>キーボードフォーカスを見つけやすくする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Use a focus indicator of sufficient size and contrast.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>十分な大きさ及びコントラストのフォーカスインジケータを使用する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Many people can't see small changes in visual appearance, including older people.</dd>
+                  <dt>なぜそれが重要か</dt>
+                  <dd>高齢者を含め、多くの人は見た目の小さな変化に気づかない。</dd>
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The purpose of this Success Criterion is to ensure a keyboard focus indicator is clearly
-                  visible and discernible. Focus Appearance is closely related to <a href="https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html">2.4.7 Focus Visible</a> and <a href="https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html">1.4.11 Non-text Contrast</a>. Focus Visible requires that a visible focus indicator exists while a component has
-                  keyboard focus; Focus Appearance defines a minimum level of visibility. Where Non-text
-                  Contrast requires a component to have adequate contrast against the background in
-                  each of its states, Focus Appearance requires sufficient contrast for the focus indicator
-                  itself.
+               <h2>意図</h2>
+               <p>この達成基準の意図は、キーボードフォーカスインジケータが明確に表示され、識別できるようにすることである。フォーカスの外観は、<a href="focus-visible.html">2.4.7 フォーカスの可視化</a>及び <a href="non-text-contrast.html">1.4.11 非テキストのコントラスト</a>と密接に関連している。フォーカスの可視化では、コンポーネントにキーボードフォーカスがある間に、可視のフォーカスインジケータを必要とする。フォーカスの外観は、可視性の最小レベルを定義する。非テキストコントラストでは、コンポーネントが各状態の背景に対して十分なコントラストを持つことを要求するが、フォーカスの外観では、フォーカスインジケータ自体に十分なコントラストを必要とする。
                </p>
-               <p>For sighted people with mobility impairments who use a keyboard or a device that utilizes
-                  the <a href="https://www.w3.org/TR/WCAG22/#dfn-keyboard-interface">keyboard interface</a> (such as a switch or voice input), knowing the current point of focus is very important.
-                  Visible focus must also meet the needs of users with low vision, who may also rely
-                  on the keyboard.
+               <p>キーボード又は<a href="#dfn-keyboard-interface">キーボードインタフェース</a>を利用するデバイス (スイッチ、音声入力など) を使用する、運動障害のある目の見える人にとって、カレントフォーカスを知ることは非常に重要である。可視フォーカスは、キーボードに依存する可能性があるロービジョンの利用者のニーズも満たさなければならない。
```

Comment:
「カレントフォーカス」(the current point of focus) という訳語が、なじみにくい感じがしたので、少し意訳気味ですが、以下、いかがでしょうか？

```suggestion
               <p>キーボード又は<a href="#dfn-keyboard-interface">キーボードインタフェース</a>を利用するデバイス (スイッチ、音声入力など) を使用する、運動障害のある目の見える人にとって、フォーカスの現在地を知ることは非常に重要である。可視フォーカスは、キーボードに依存する可能性があるロービジョンの利用者のニーズも満たさなければならない。
```